README.md

```diff
@@ -1,30 +1,26 @@
 # Zed [![Tests][tests-img]][tests]
 
-The Zed project is a new, clean-slate design for a data engineering stack.
-At Zed's foundation lies a new family of self-describing
-data formats based on the "Zed data model", which blends the highly structured
-approach of dataframes and relational tables with the loosely structured
-document model of JSON.
+Zed is an open-source, cloud-native, and searchable data lake for
+semi-structured data.
+
+Zed lakes utilize a superset of the relational and JSON document data models,
+but you do not need to create tables or schemas ahead of time to add data.
+
+At Zed's foundation lies a new family of self-describing data formats based on the
+[Zed data model](https://github.com/brimdata/zed/blob/main/docs/formats/zson.md#1-introduction),
+which unifies the highly structured approach of dataframes and relational tables
+with the loosely structured document model of JSON.
 
 While the Zed system is built around its family of data formats, it is also
 interoperable with popular data formats like CSV, (ND)JSON, and Parquet.
 
 This repository contains tools and components used to organize, search, analyze,
 and store Zed data, including:
 
-* The [zq](cmd/zq/README.md) command line tool for searching and analyzing data
-* The [zqd](ppl/cmd/zqd/README.md) daemon, which serves a REST API to manage
- and query Zed data lakes, and is the backend for the [Brim](https://github.com/brimdata/brim)
- application
-* The [zapi](cmd/zapi/README.md) command line tool, for interacting with the
-API provided by zqd
+* The [zed](cmd/zed/README.md) command line tool managing, searching, and querying a Zed lake
 * The [Zed language](docs/language/README.md) documentation
 * The [Zed formats](docs/formats/README.md) specifications and documentation
 
-We believe the Zed data architecture provides a powerful foundation for the
-modern data lake and are actively developing tools and software components
-for the emerging "Zed data lake".
-
 ## Installation
 
 To install `zq` or any other tool from this repo, you can either clone the repo
```

Comment:
```suggestion
To install [`zq`](cmd/zed/README.md#zq) or any other tool from this repo, you can either clone the repo
```

There's enough links out there still potentially pointing to [`https://github.com/brimsec/zq`](https://github.com/brimsec/zq) (and users that found what's-now-known-as-Zed _because_ of `zq`) that I'm in favor of maintaining _some_ mention of `zq` here, and I think the added link here helps that just a bit, since legacy users will not immediately "get" that `zed query` is what used to be known as `zq`, that the separate `zq` binary still exists though, etc.

Personally, I'd actually prefer instead mentioning `zq` briefly in the bulleted line above, e.g.

> The \[zed\](cmd/zed/README.md) command line tool (which includes `` `zq`](cmd/zed/README.md#zq))`` for managing, searching, and querying a Zed lake

That is, in the past we had at least some success with branding of `zq` as being something like an improvement vs. `jq`, so people who find their way here looking for it might appreciate finding it mentioned prominently. But I also understand the purity push to emphasize all things "lake" and how that might mean leaving behind some prior branding, so I'm prepared to go with the flow.

Reply:
This is transitional.  I will add a sentence after the bullets that clarifies `zq` is a command-line shortcut for `zed query`.